nbconvert/filters/markdown_mistune.py

```diff
@@ -53,8 +96,14 @@ class MarkdownWithMath(mistune.Markdown):
     def __init__(self, renderer, **kwargs):
         if 'inline' not in kwargs:
             kwargs['inline'] = MathInlineLexer
+        if 'block' not in kwargs:
+            kwargs['block'] = MathBlockLexer
         super(MarkdownWithMath, self).__init__(renderer, **kwargs)
 
+    
+    def output_multiline_math(self):
+        return self.inline(self.token["text"])
```

Comment:
Does this mean we're parsing it again with the inline parser? Can we skip that, given that we just want to put it in the output unmodified?

Reply:
We're parsing it semantically for the first time in the inline parser. We actually aren't putting it in the output unmodified, because we need to escape the individual parts of it separately which occurs in the rendering step.

Reply:
Gotcha, thanks.